util/schemas.js

```diff
@@ -100,7 +100,7 @@ const configSchema = joi.object({
 	"server": joi.object({
 		"host": joi.string().hostname().default("connect.2b2t.org")
 			.description("Address of the server to connect to"),
-		"version": joi.string().default("1.12.2") // to-do: regex idea: must start with a number and contain only numbers & decimals?
+		"version": joi.string().pattern(/^[0-9.]+$/).default("1.12.2") 
```

Comment:
LGTM. Just to put it out there, this could be improved in a future commit with [Mineflayer's supportedVersions array](https://github.com/PrismarineJS/mineflayer/blob/master/lib/version.js).